app/views/content/life-as-a-teacher/age-groups-and-specialisms/primary/_article.html.erb

```diff
@@ -0,0 +1,91 @@
+   <p>The areas of learning pupils will cover in recpetion include:</p>
+   <ul>
+    <li>communication and language</li>
+    <li>personal, social and emotional development</li>
+    <li>physical development</li>
+    <li>literacy</li>
+    <li>mathematics</li>
+    <li>understanding the world</li>
+    <li>expressive arts and design</li>
+  </ul>
+  <p>At the end of reception every child will be observed and assessed against the early learning goals found in the <a href="https://www.gov.uk/government/publications/early-years-foundation-stage-framework--2">early years framework</a>.</p>
+  <h3>Key stage 1 (KS1)</h3>
+  <p>KS1 covers years 1 and 2 with pupils aged between 4 and 7. At this age, pupils will build on skills in maths, reading and writing and will transition away from play based learning. As their class teacher, you will be responsible for delivering a broad range of lessons to pupils.</p>
+  <p>Schools who follow the national curriculum for KS1 will have to teach:</p>
+    <ul>
+      <li>English</li>
+      <li>maths</li>
+      <li>science</li>
+      <li>design and technology</li>
+      <li>history</li>
+      <li>geography</li>
+      <li>art and design</li>
+      <li>physical education</li>
+      <li>computing</li>
+    </ul>
+    <p> At the end of year 1, pupils will take a phonics screening test where they will be asked to read from a list of 40 words to check their ability to read.</p>
+    <p>At the end of year 2, pupils will sit a more formal exam in English and maths to check what they have learned so far.</p>
+  <h3>Key stage 2 (KS2)</h3>
+  <p>KS2 is years 3 to 6 with pupils aged between 7 and 11. In key stage 2, pupils will build on the skills that they learned in KS1 and be expected to become increasingly independent. As their class teacher you will be responsible for teaching pupils all subjects.</p>
+    <p>Schools following the national curriculum for KS2 have to teach: </p>
+    <ul>
+      <li>English</li>
+      <li>maths</li>
+      <li>science</li>
+      <li>design and technology</li>
+      <li>history</li>
+      <li>geography</li>
+      <li>art and design</li>
+      <li>physical education</li>
+      <li>computing</li>
+    </ul>
+    <p>At the end of year 4, pupils will take a multiplication tables check to see if they have learned the skills which will help them with increasingly challenging maths.</p>
+    <p>At the end of KS2, pupils will sit national assessments in maths, science, English reading and writing. They will then transition to secondary school. This is an exciting stage for pupils as they will be ready to take on the new challeneges of increasing independence and more in depth studies.</p>
+
+    <h2 class="heading--box-blue">Pupils with special educational needs and disabilities (SEND)</h2>
```

Comment:
Should this bit be a partial?

Reply:
Hey, yeah I will swap it for a partial when the secondary page has gone up